### PR TITLE
Host Metrics for the Container Agent

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Rework Getting Further with Docker

### Motivation
FAQ Audit

### Preview link
https://docs-staging.datadoghq.com/ruth/further-with-docker/agent/faq/host-metrics-with-the-container-agent/

### CODE REVIEW

File: content/en/agent/faq/host-metrics-with-the-container-agent.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  text: "Learn more about the Datadog Docker Agent"
+---
+
+## Disk check
+The [Disk check][1] reports host metrics with the container Agent for any storage mounted on the host and exposed to the container as a volume. No additional setup is required.
+
+### Metrics for partitions
+The layered nature of Linux storage (block devices, logical volumes, and partitions) makes it necessary to have a partition mounted to report its free space.
+
+The container Agent reports disk metrics and rates for every partition that is accessible (even if only partially). This separation is enforced by cgroups and Docker. To allow disk usage reporting on a partition, you need to expose it through a Docker volume with the `-v` argument to `docker run`. The following options are available:
```
Comment (apigirl) [inline content/en/agent/faq/host-metrics-with-the-container-agent.md]:
```suggestion
The container Agent reports disk metrics and rates for every wholly or partially accessible partition. Cgroups and Docker enforce this separation. To allow disk usage reporting on a partition, you need to expose it through a Docker volume with the `-v` argument to `docker run`. The following options are available:
```

File: content/en/agent/faq/host-metrics-with-the-container-agent.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    ```
+
+* If the mount path is unknown (dynamically mounted volumes), but the parent directory is constant, expose the parent folder. The Agent is able to access all volumes mounted in this folder's children.
+    ```
+    -v /mnt/:/host/mnt:ro
+    ```
+
+### Troubleshooting
+#### Missing disk metrics
+If you customized the Docker image or mount a custom directory to the Agent's `conf.d` folder, make sure the Disk check's default `conf.yaml` (or a customized `conf.yaml`) is present, or the check is disabled.
```
Comment (apigirl) [inline content/en/agent/faq/host-metrics-with-the-container-agent.md]:
```suggestion
If you customized the Docker image or mount a custom directory to the Agent's `conf.d` folder, make sure the Disk check's default `conf.yaml` (or a customized `conf.yaml`) is present to enable the check.
```

I am not sure whether this is saying either to verify the conf.yaml or disable the check, or whether it's saying to if there isn't a conf.yaml, it causes the check to be disabled. I assumed the latter with this suggestion.

Comment (ruthnaebeck) [reply content/en/agent/faq/host-metrics-with-the-container-agent.md]: I will re-word

Comment (ruthnaebeck) [inline content/en/agent/faq/host-metrics-with-the-container-agent.md]:
```suggestion
If you customized the Docker image or mount a custom directory to the Agent's `conf.d` folder, do one of the following for the Disk check:

- Make sure the default `conf.yaml` is present.
- Enable a customized `conf.yaml`.
- Disable the check.
```

**Pull request merged**